README.md

```diff
@@ -41,6 +41,11 @@ or you can read this documentation inside vscode with the command `Open XML Docu
 
 ## Requirements
 
+For running the binary version:
+  * Windows, macOS, or Linux, on a x86_64 CPU
+    * We don't make a binary specific for Apple ARM (Apple Silicon), and we cannot test the x86_64 binary on an Apple ARM system currently. However, it might work using the translation layer.
```

Comment:
We don't make a binary specific for Apple ARM (Apple Silicon), but the x86_64 binary seems to work correctly via [Rosetta 2](https://support.apple.com/en-us/HT211861).